.github/workflows/translate-ck3.yml

```
@@ -48,7 +48,13 @@ jobs:
         with:
           commit_options: '--signoff'
           commit_message: "chore(ck3): 번역 파일 업데이트 [skip ci]"
-          commit_user_name: 'github-actions[bot]'
-          commit_user_email: '41898282+github-actions[bot]@users.noreply.github.com'
-          commit_author: 'dungsil <39783731+dungsil@users.noreply.github.com>'
-    
+          pull_options: '--rebase --autostash'
+      - name: Create issue for untranslated items
+        if: always()
+        uses: ./.github/actions/create-untranslated-issues
```

actionlint (1.7.9)

missing input "github-token" which is required by action "Create Untranslated Items Issues" defined at "./.github/actions/create-untranslated-issues". all required inputs are "game", "github-token"

(action)